server/src/Application/Validation/Validators/Flashcard/UpdateFlashcardRequestValidator.cs

```diff
@@ -0,0 +1,24 @@
+using FluentValidation;
+using StudyZen.Application.Dtos;
+using StudyZen.Application.Services;
+using StudyZen.Application.Validation.Validators;
+using StudyZen.Domain.Entities;
+
+namespace StudyZen.Application.Validators;
+
+public class UpdateFlashcardRequestValidator : BaseValidator<UpdateFlashcardDto>
+{
+    public UpdateFlashcardRequestValidator()
+    {
+        RuleFor(flashcard => flashcard.Question)
+        .Must(NullOrNotWhiteSpace)
```

Comment:
Why not use NotEmpty?

Reply:
because Question in the patch request can be null when it is not intended to be updated. NotEmpty throws an exception when the field is empty, null or whitespace